### ISSUE TITLE
Resizing crash with information tab

### ISSUE THREAD

**Issue by BitKnowsTech** [MatthewCech/ASCIIPlayer#1]
when the `i` information menu is up and the height of the display is resized to zero the program returns a `HEAP CORRUPTION DETECTED` error. causing it to crash spectacularly.

**Comment by MatthewCech** [comment]
Problem addressed in the upcoming spectrum_visualizer branch - will be merged into develop and subsequently the next release! I'll close this issue once the fix is merged to master.
